gpic/gpic_test.go

```diff
@@ -52,13 +52,25 @@ func TestNewAvatar(t *testing.T) {
 		// Uses a special fake token designed just for tests, first a bad one, then a good one
 		// sample 14
 		{[]string{"ghp_test-broken-token", "ghp_test-working-token"}, 0, "", true},
+		// GitLab username
+		// sample 15
+		{[]string{"gl:chayev"}, 0, "https://gitlab.com/uploads/-/system/user/avatar/5945977/avatar.png", true},
+		// Invalid GitLab username
+		// sample 16
+		{[]string{"gl:chayev1"}, -1, "", false},
```

Comment:
This test case can't be used. It can't be assumed that this will always be an invalid username.

Reply:
What username should be passed as an invalid one then?

Reply:
Nothing? I didn't do one for GitHub. Or if you really want to use one, maybe find GitLab's rules for usernames and then come up with one that would be impossible to use based on those rules.

Reply:
changed it to "chayev.png" as per the gilab docs: https://docs.gitlab.com/ee/user/profile/#change-your-username